src/main/java/wolox/training/TrainingApplication.java

```diff
@@ -0,0 +1,13 @@
+package wolox.training;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class TrainingApplication {
+
+	public static void main(String[] args) {
```

Comment:
We recommend using 4 spaces instead of a tab character, you can configure this in your IDE preferences:  
![image](https://user-images.githubusercontent.com/43755493/90777352-ac996e00-e2d1-11ea-8f3b-1d3374fdc31c.png)

Let us know if you need any help 😄

Reply:
![image](https://user-images.githubusercontent.com/69865427/90784953-70f9a680-e2c7-11ea-8b9f-75f6b8875f3b.png)
Thanks, i have edited to right mode